src/main/java/com/fundynamic/d2tm/game/controls/Mouse.java

```diff
@@ -12,12 +14,18 @@
  */
 public class Mouse {
 
+    private Player playerControlled;
```

Comment:
`playerControlled` sounds like a boolean value. I like the name `controllingPlayer` better.